frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -5,6 +5,46 @@
     </div>
     <div class="card-body">
       <form>
+        <div class="row">
+          <div class="col">
+            <input
+              v-model="over13Checked"
+              type="checkbox"
+              @change="updateDateRestriction"
+            />
```

Comment:
```suggestion
            <input
              v-model="over13Checked"
              type="checkbox"
              @change="updateDateRestriction"
              data-over-13-checkbox
            />
```

Agrégale ese data-attribute para poder seleccionar al checkbox desde Cypress.